api/client/diff.go

```diff
@@ -34,7 +34,7 @@ func (cli *DockerCli) CmdDiff(args ...string) error {
 		var kind string
 		switch change.Kind {
 		case archive.ChangeModify:
-			kind = "C"
+			kind = "M"
```

Comment:
I meant that by changing this, people may have implemented scripts that use these characters. i.e. the output will change from

```
A /foobar
A /foobaz
C /root
A /root/.bash_history
C /run
A /run/nginx.pid
C /var
C /var/cache
C /var/cache/nginx
A /var/cache/nginx/client_temp
A /var/cache/nginx/fastcgi_temp
A /var/cache/nginx/proxy_temp
A /var/cache/nginx/scgi_temp
A /var/cache/nginx/uwsgi_temp
C /var/log
C /var/log/lastlog
```

to

```
A /foobar
A /foobaz
M /root
A /root/.bash_history
M /run
A /run/nginx.pid
M /var
M /var/cache
M /var/cache/nginx
A /var/cache/nginx/client_temp
A /var/cache/nginx/fastcgi_temp
A /var/cache/nginx/proxy_temp
A /var/cache/nginx/scgi_temp
A /var/cache/nginx/uwsgi_temp
M /var/log
M /var/log/lastlog
```